### PR TITLE
Mute default :focus form highlight

### PR DESCRIPTION
**Pull request by josuke0227**
The applied border colour (border-color: #000;) is cascaded by the default border highlight (blue colour in Chrome). Therefore I reckon the default styling needs to be muted.

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks - I am closing this because I _think_ it is invalid. There don't appear to be any `outline` properties set on the box either before or after this change, and as far as I can tell there isn't any visual change either. 

If you can show exactly what the before and after change is (screenshots) then perhaps we can discuss. Note that any changes mae here also have to be made in the corresponding example code linked from the page.

**Comment by josuke0227** [comment]
Thanks for your response.

Here's where I reproduced the change:
https://developer.mozilla.org/en-US/docs/Learn/Forms/Your_first_form#summary

Here's how I applied the change and confirmed both the property is applied to the element and the appearance is affected as per the property.

Reporoduction_process1
Reproduction_result2
Result

**Comment by hamishwillee** [comment]
Thanks you, but I cannot reproduce. Using the current version you can see below the border is black to start with and that there is no computed property for outline - i.e. no blue outline that is inherited. It may be that this is set differently on another browser, but I still think adding this fix would be more confusing than helpful. 

image

**Comment by josuke0227** [comment]
Hi, I tried to see how the focused form's appearance changes using my Windows laptop and I've got the same outline as yours so the difference in border colour can be concluded in the difference of the OS.

I also found the style (either blue or black) is defined in ":forcus-visible" pseudo selector.
Here is the evidence:

Process1

Your "outline-color" might be "rgb(16, 16, 16)"
Result

From above,  if you need to change the border colour to #000, you need the "outline: none" to make the border colour visible. However, it makes sense that it will be confusing as you mentioned.  What about style ling like below?
input:focus,
textarea:focus {
  /* To give a little highlight on active elements */
  outline-color: #000;
}

**Comment by hamishwillee** [comment]
THanks for your patience @josuke0227 .

Yes, I think that instead of muting the outline, what you have suggested will work better, and more important, be more consistent across browsers.

```css
input:focus,
textarea:focus {
/* To give a little highlight on active elements */
outline-color: #000;
}
```

BUT, as I noted before, we also need to make this same change in https://github.com/mdn/learning-area/blob/main/html/forms/your-first-HTML-form/first-form-styled.html and the other linked examples - can you do that too, and link to here?

This should be tested on at least chrome and FF.

**Comment by hamishwillee** [comment]
@josuke0227 - thank you for the testing and confirmation. 

Can you apply your fix here, and cross link from your PR to https://github.com/mdn/learning-area/blob/main/html/forms/your-first-HTML-form/first-form-styled.html

I can then review and merge them together.

**Comment by josuke0227** [comment]
Hi @hamishwillee,
Thanks for checking the results.

I've just updated the other one.
Please check out the link below.
[https://github.com/mdn/learning-area/pull/767](https://github.com/mdn/learning-area/pull/767)

Thank you!

**Comment by hamishwillee** [comment]
Thanks @josuke0227 - might not get to this until Friday, but have not forgotten!

File: files/en-us/learn/forms/your_first_form/index.md [hunk 1]
```diff
@@ -225,6 +225,8 @@ textarea {
 
 input:focus,
 textarea:focus {
+  /* Mute default highlight */
+  outline: none;
   /* Additional highlight for focused elements */
   border-color: #000;
 }
```
Comment (hamishwillee) [inline files/en-us/learn/forms/your_first_form/index.md:232]: This is what was agreed in thread.

```suggestion
textarea:focus {
  /* To give a little highlight on active elements */
  outline-color: #000;
}
```

File: files/en-us/learn/forms/your_first_form/index.md [hunk 2]
```diff
@@ -225,8 +225,10 @@ textarea {
 
 input:focus,
 textarea:focus {
-  /* Additional highlight for focused elements */
-  border-color: #000;
+  /* Set the outline width and style /
```
Comment (hamishwillee) [inline files/en-us/learn/forms/your_first_form/index.md:228]:
```suggestion
  /* Set the outline width and style */
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much @josuke0227 - also for your patience with my review.

**Pull request merged**